Automattic/jetpack
# Fix out-of-bounds submenus on the admin menu

### Specification

This PR aims to fix the positioning of the submenus in the admin menu. It does that by injecting the core's mobile toggler with ID `wp-admin-bar-menu-toggle` to enable the menu adjustment inside the core's `common.js` file.

More information at pdqkbG-2eG-p2

Fixes https://github.com/Automattic/wp-calypso/issues/71198

### Other information:


## Testing instructions:






* Navigate to a wp-admin page with content outside the viewport, or resize your window.
* Notice the, e.g., the Settings submenu being rendered outside of the viewport.
* Apply this patch
* Ensure that the submenu is rendered inside the viewport.

Note that I didn't test this in Simple sites as I couldn't find the proper way of doing so. Please consider testing this patch in both Simple and AT sites.

## Does this pull request change what data or activity we track or use?

No.
